docs/docs/how-to/functions/getting-started.md

```diff
@@ -123,4 +123,65 @@ export default async function postNewPersonHandler(req, res) {
 
 ## Forms
 
-Forms and Functions are often used together. See the [basic form example](https://github.com/gatsbyjs/gatsby-functions-beta/tree/main/examples/basic-form) as well as the [Forms doc page](/docs/how-to/adding-common-features/adding-forms/) which walks you through building a custom form.
+Forms and Functions are often used together. For a working example you can play with locally, see the [form example](https://github.com/gatsbyjs/gatsby-functions-beta/tree/main/examples/basic-form). The [Forms doc page](/docs/how-to/adding-common-features/adding-forms/) is a gentle introduction for building forms in React. Below is sample code for a very simple form that submits to a function that you can use as a basis for building out forms in Gatsby.
+
+```js:title=src/api/form.js
+export default function formHandler(req, res) {
+  // req.body has the form values
+  console.log(req.body);
```

Comment:
Could show a super basic example of validation, e.g.

```
if (!req.body.name) {
  return res.status(422).json({
    message: 'Name field is required'
  })
}
```

Reply:
👍